runtime/jvmti/jvmtiHook.c

```diff
@@ -2575,7 +2575,7 @@ lookupNativeAddressHelper(J9VMThread *currentThread, J9JVMTIData *jvmtiData, J9M
 	U_8 *nameData = J9UTF8_DATA(methodName);
 	UDATA nameLength = J9UTF8_LENGTH(methodName);
 
-	Trc_JVMTI_lookupNativeAddressHelper_Entry(currentThread, nativeMethod,
+	Trc_JVMTI_lookupNativeAddressHelper_Entry2(currentThread, nativeMethod,
```

Comment:
You don't need to change the tracepoint name; if we need to keep the old one, rename it (and mark it obsolete): the code can use the same name.

Reply:
The change is made in existing tracepoints, hence no need to modify the code anyway.